rpm_spec/subpackages/manageiq-ui

```diff
@@ -3,6 +3,7 @@ Summary: %{product_summary} UI
 Requires: %{name}-core = %{version}-%{release}
 Requires: httpd
 Requires: mod_ssl
+BuildRequires: nodejs
```

Comment:
sort of ditto, but in the gemset the BuildRequires come before teh Requires, so let's do the same here.